packages/vm/tests/tester/runners/GeneralStateTestsRunner.ts

```diff
@@ -116,20 +117,10 @@ async function runTestCase(options: any, testData: any, t: tape.Test) {
           t.comment(JSON.stringify(stateRoot))
         })
       }
-
       try {
         await vm.runTx({ tx, block })
         execInfo = 'successful tx run'
       } catch (e: any) {
-        // If tx is invalid and coinbase is empty, the test harness
-        // expects the coinbase account to be deleted from state.
-        // Without this ecmul_0-3_5616_28000_96 would fail.
-        const account = await vm.stateManager.getAccount(block.header.coinbase)
-        if (account.balance.isZero()) {
-          await vm.stateManager.putAccount(block.header.coinbase, new Account())
-          await vm.stateManager.cleanupTouchedAccounts()
-          await vm.stateManager._cache.flush()
-        }
```

Comment:
Why is this not being cleared by a `revert()` in `VM.runTx()` when the tx fails? Should be the case I guess? 🤔

Reply:
It is now.  @ryanio  and I looked at this and this looks to be some workaround for presumably [this test[(https://github.com/ethereum/tests/blob/33c3174e88fa2315a316602007393ee5180bf851/src/GeneralStateTestsFiller/stZeroKnowledge2/ecmul_0-3_5616_28000_96Filler.json) based on the comments but the code itself is quite old and it was mutating the state after the transaction failed and was causing [this new set of tests for EIP-3607](https://github.com/ethereum/tests/tree/develop/GeneralStateTests/stEIP3607) to fail.  After removing it, I've re-run the tests the workaround was for and they are passing so I think this just became duplicative code that wasn't failing any tests.

Reply:
Nice 👍